tst/blackbox_repository/test_blackbox.py

```diff
@@ -138,19 +139,22 @@ def test_blackbox_seed():
 
 
 def test_blackbox_offline_serialization():
-    y = x1 * x2
-    data = np.stack([x1, x2, y]).T
-    df = pd.DataFrame(data=data, columns=["hp_x1", "hp_x2", "metric_rmse"])
+    try:
```

Comment:
This would be undesirable as the test will silently fail if this dependency is not installed, could you revert it? (also holds for the unit-test bellow)